### PR TITLE
Contributing section in README

### PR DESCRIPTION
**Pull request by gi114**
Adding Contributing notes on submodules, push access and raising PRs for future contributors

### CODE REVIEW

**Comment by glatard** [comment]
Hi @gi114, thanks for working on this! A few comments:
* This should be merged in https://github.com/CONP-PCNO/conp-dataset/blob/master/.github/CONTRIBUTING.md
* I don't think that https://github.com/conpdatasets should be the default organization. Some datasets, such as the ones in https://github.com/CONP-PCNO/conp-dataset/tree/master/investigators/Khanlab are not there and that's totally fine.
* https://github.com/conpdatasets is surely not a fork of an OpenNeuro dataset. It looks like the paragraph mentioning this is specific to a particular dataset.

File: README.md [hunk 1]
```diff
@@ -48,3 +48,11 @@ You can also search for relevant files and sub-datasets as follows:
 ```console
 datalad search T1
 ```
+
+## Contributing
+
+This repository has multiple git submodules pointing at dataset instances available in the [conpdatasets](https://github.com/conpdatasets) repository.
```
Comment (dlq) [inline README.md]: Is this the right CONP datasets repository?

Comment (dlq) [reply README.md]: It is.

**Comment by gi114** [comment]
Hi @glatard, thanks you for your comments. I wrote this notes thinking that conpdatasets is the default organization with its datasets forked from OpenNeuroDatasets. My mistake, I will delete the second part. Should I leave the first note on push access? If yes, I will add it to CONTRIBUTING.md

**Comment by glatard** [comment]
Hi @gi114, push access is not needed but also not desirable: we want all contributions to go through Pull Requests so that they can be tested and vetted.

**Comment by gi114** [comment]
Ok I see, I will close the PR then as there is nothing else to it.

File: README.md [hunk 2]
```diff
@@ -48,3 +48,11 @@ You can also search for relevant files and sub-datasets as follows:
...
+Therefore, to contribute to such datasets it is required that collaborators have push access to both repositories.
```
Comment (PapillonMcGill) [inline README.md]: Not needed, can issue a PR the conp-dataset. It will be merge if acceptable

File: README.md [hunk 3]
```diff
@@ -48,3 +48,11 @@ You can also search for relevant files and sub-datasets as follows:
...
+
+In addition, it is important to keep in mind that the conpdatasets repository is a fork of the [OpenNeuroDatasets](https://github.com/OpenNeuroDatasets/ds001634) project.
```
Comment (PapillonMcGill) [inline README.md]: Not true for all of the datasets.
Prevent-AD is not on OpenNeuro.
Same for Multicenter-phantom.

Comment (gi114) [reply README.md]: Hi @PapillonMcGill, thank you for your comment, I will close the PR for the moment, as this information as you say may not be correct.

**Pull request closed without merging**